content/release-notes/2023-11-23-docs.md

```diff
@@ -0,0 +1,11 @@
+---
+description: Read about the latest product updates from Neon
```

Comment:
If the postgres docs are the only update, consider putting that in the description instead:

Neon now includes a mirror of the PostgeSQL documentation, with integrated search, in our docs.

Reply:
sorry, I missed this!